<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>116</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Scheduler GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/pics/pics/star.png</normaloff>:/pics/pics/star.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget, QMenuBar::item {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #c5d8ef, stop: 1 #89a5c3);
}

QLabel, QSlider {
	background-color: transparent;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Launch search</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progress">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>308</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_Project"/>
    <addaction name="actionSave_Project_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuMethod">
    <property name="title">
     <string>Method</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionReset"/>
    <addaction name="actionLoad_New_Method"/>
    <addaction name="actionSettings"/>
    <addaction name="actionLaunch_Viewer"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionLanguage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionContents"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionChange_name"/>
    <addaction name="actionChange_Limits"/>
    <addaction name="actionLoad_New_System"/>
    <addaction name="actionGenerate_Random_System"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionTrace"/>
    <addaction name="actionResult"/>
    <addaction name="actionGenerate_Code"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProject"/>
   <addaction name="menuMethod"/>
   <addaction name="menuWindow"/>
   <addaction name="menuExport"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Project"/>
   <addaction name="actionOpen_Project"/>
   <addaction name="actionSave_Project"/>
   <addaction name="actionLaunch_Viewer"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Scheduler</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset>
     <normalon>:/pics/pics/page.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset>
     <normalon>:/pics/pics/folder.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="icon">
    <iconset>
     <normalon>:/pics/pics/cd.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_Project_As">
   <property name="text">
    <string>Save Project As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>Contents...</string>
   </property>
  </action>
  <action name="actionAbout_Scheduler">
   <property name="text">
    <string>About Scheduler</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="actionChange_name">
   <property name="text">
    <string>Change Name...</string>
   </property>
  </action>
  <action name="actionLoad_New_System">
   <property name="text">
    <string>Load New System...</string>
   </property>
  </action>
  <action name="actionLoad_New_Method">
   <property name="text">
    <string>Load New Method...</string>
   </property>
  </action>
  <action name="actionGenerate_Random_System">
   <property name="text">
    <string>Generate Random System</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>Language...</string>
   </property>
  </action>
  <action name="actionChange_Limits">
   <property name="text">
    <string>Change Limits...</string>
   </property>
  </action>
  <action name="actionLaunch_Viewer">
   <property name="icon">
    <iconset>
     <normalon>:/pics/pics/chart.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Launch Viewer</string>
   </property>
  </action>
  <action name="actionTrace">
   <property name="text">
    <string>Trace...</string>
   </property>
  </action>
  <action name="actionResult">
   <property name="text">
    <string>Result...</string>
   </property>
  </action>
  <action name="actionGenerate_Code">
   <property name="text">
    <string>Generate Code...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>Exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>About()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_Project</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>NewProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Project</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_Project</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>SaveProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_Project_As</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>SaveProjectAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStart</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>Run()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReset</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ResetSchedule()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGenerate_Random_System</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>GenerateRandomSystem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad_New_System</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>LoadSystem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad_New_Method</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>LoadMethod()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChange_name</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ChangeName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>Settings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLanguage</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ChangeLanguage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChange_Limits</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ChangeLimits()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLaunch_Viewer</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>LaunchViewer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTrace</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ExportTrace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Exit()</slot>
  <slot>About()</slot>
  <slot>NewProject()</slot>
  <slot>OpenProject()</slot>
  <slot>SaveProject()</slot>
  <slot>SaveProjectAs()</slot>
  <slot>Run()</slot>
  <slot>Trace()</slot>
  <slot>StepForward()</slot>
  <slot>StepBackward()</slot>
  <slot>ChangeScale(int)</slot>
  <slot>SelectSchedule(QString)</slot>
  <slot>ResetSchedule()</slot>
  <slot>ShowLastStep()</slot>
  <slot>Preferences()</slot>
  <slot>GenerateRandomSystem()</slot>
  <slot>LoadSystem()</slot>
  <slot>LoadMethod()</slot>
  <slot>ChangeName()</slot>
  <slot>Settings()</slot>
  <slot>ChangeLanguage()</slot>
  <slot>ChangeLimits()</slot>
  <slot>LaunchViewer()</slot>
  <slot>ExportTrace()</slot>
 </slots>
</ui>
